<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadDataDialog</class>
 <widget class="QDialog" name="LoadDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка данных</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="editC3"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelRays">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Файл параметров излучения</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonRays">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonC1">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="buttonC2">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="buttonC3">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelC3">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Файл С3</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="editC1"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelC1">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Файл C1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="editC2"/>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelC2">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Файл С2</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="editRays"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
